### ISSUE TITLE
[RTL] Filters

### ISSUE THREAD

**Issue by Siemienik** [handsontable/handsontable#8828]
### Description

For RTL, the filters menu should:
* should be fixed after #8826 
* Content of filters menu should be aligned to the right
* Checkboxes should be on the right of values items
* Select All / Clear should be on the left
* Ok / Cancel button should be swapped
* Arrows in dropdown menus should be on the left

### How to reproduce

1. use this [configuration](https://github.com/Siemienik/handsontable-storybooks/blob/28767986ae46e6525f45d6c11d77e575b81f0a34/vanilla/configurations.js#L771-L792)
```js
{
    data: [
        ['Lorem', 'ipsum', 'dolor', 'sit', '12/1/2015', 23],
        ['adipiscing', 'elit', 'Ut', 'imperdiet', '5/12/2015', 6],
        ['Pellentesque', 'vulputate', 'leo', 'semper', '10/23/2015', 26],
        ['diam', 'et', 'malesuada', 'libero', '12/1/2014', 98],
        ['orci', 'et', 'dignissim', 'hendrerit', '12/1/2016', 8.5]
    ],
    columns: [
        { type: 'text' },
        { type: 'text' },
        { type: 'text' },
        { type: 'text' },
        { type: 'date', dateFormat: 'M/D/YYYY' },
        { type: 'numeric' }
    ],
    colHeaders: true,
    dropdownMenu: true,
    filters: true,
    height: 'auto',
    licenseKey: 'non-commercial-and-evaluation'
}
```
2. Open the dropdown menu on column E
3. result:

![image](https://user-images.githubusercontent.com/2893099/135973468-3bdb6ead-094d-4a92-bf64-fc95c18fd33d.png)

**Comment by aninde** [comment]
Fixed in v12.0.0 https://jsfiddle.net/t69meyL0/1/

**Comment by AMBudnik** [comment]
Closed with https://github.com/handsontable/handsontable/issues/8760 (RTL Support), published in Handsontable v12.

For more information please check out our Release Notes - https://handsontable.com/docs/release-notes/#_12-0-0
